RevenueCatUI/Templates/V2/Previews/TemplateComponentsViewPreviews/ButtonWithFooterPreview.swift

```diff
@@ -92,6 +92,7 @@ private enum ButtonWithSheetPreview {
     static let package = PaywallComponent.PackageComponent(
         packageID: "weekly",
         isSelectedByDefault: false,
+        applePromoOfferProductCode: nil,
```

Comment:
I guess that `promoOffer` is already an "Apple" concept so I wonder if we should remove the `apple` prefix... But then again, I'm not opposed to make it more explicit